lib/py/images/images.py

```diff
@@ -390,7 +390,8 @@ def dump(self, extra, f, pbuff):
         if len(s) != entry['nsems']:
             raise Exception("Number of semaphores mismatch")
         f.write(s.tobytes())
-        f.write('\0' * (rounded - size))
+        end = '\0' * (rounded - size)
```

Comment:
b'\0' * (rounded - size) - will do the same, will it not?

Reply:
Yes. Changed.